api/system-backup/execute

```diff
@@ -47,7 +54,8 @@ function _restore
     echo '{"progress":"0.50","time":"2.00","exit":0,"event":"restore-config","state":"done","step":1,"pid":"","action":"Pulling from history"}';
 
     echo '{"step":2,"pid":"","action":"Restoring configuration","event":"restore-config","state":"running"}'
-    /sbin/e-smith/restore-config $remap &>/dev/null
+    echo "{\"status\": \"/sbin/e-smith/restore-config $remap $reinstall\"}"
+    /sbin/e-smith/restore-config $remap $reinstall &>/dev/null
```

Comment:
Can we preserve stderr?

```suggestion
    /sbin/e-smith/restore-config $remap $reinstall >/dev/null
```

Reply:
If stderr shouldn't be catch, please be sure that the UI doesn't break.

Reply:
I guess stderr is drained to system journal